dist/index.js

```diff
@@ -1,19 +0,0 @@
-'use strict';
```

Comment:
I removed the `dist` package altogether. Our other node projects didn't have it checked in...

Reply:
This is correct, it should be git ignored.

It should also probably be called `lib/` for libraries.  This is what our new cookiecutter does.